script.js

```diff
@@ -0,0 +1,150 @@
+const toDoForm = document.querySelector(".toDoForm");
+const toDoInput = toDoForm.querySelector("input");
+const toDoList = document.querySelector(".toDoList");
+const finishedList = document.querySelector(".finishedList");
+const lenTodo = document.getElementById("todo-count");
+const lenFin = document.getElementById("fin-count");
+
+const TODODATA = "todoData"
+const FINDATA = "FinData"
+
+let toDoArr = [];
+let finArr = [];
+
+function loadDATA(){
+    const loadedTODO = localStorage.getItem(TODODATA);
+    const loadedFIN = localStorage.getItem(FINDATA);
+    
+    if (loadedTODO !== null) {
+        const parsedTODO = JSON.parse(loadedTODO); // parse, 가져온 것을 자바스크립트 object로 변환시켜줌
+        parsedTODO.forEach(function(input) {
+            printData(input.text,parseInt(input.id),true)
+        });
+    }
+    if (loadedFIN !== null) {
+        const parsedFIN = JSON.parse(loadedFIN); // parse, 가져온 것을 자바스크립트 object로 변환시켜줌
+        parsedFIN.forEach(function(input) {
+            printData(input.text,parseInt(input.id),false)
+        });
+    }
+}
+
+function saveDATA(){
```

Comment:
여기서 말하는 `DATA`가 `localStorage`에 저장된 데이터인지, 지금 가지고 있는 `arr`에 존재하는 데이터인지 혼동의 여지가 있네요. 데이터의 출처를 정확히 써주면 혼동을 방지할 수 있겠죠?
```suggestion
function saveDataToLocalStorage(){
```